modules/ptp-subscribing-consumer-app-to-events.adoc

```diff
@@ -1,81 +1,32 @@
 // Module included in the following assemblies:
 //
+// * networking/ptp/ptp-cloud-events-consumer-dev-reference-v2.adoc
 // * networking/ptp/ptp-cloud-events-consumer-dev-reference.adoc
 
 :_mod-docs-content-type: REFERENCE
-[id="ptp-subscribing-consumer-app-to-events_{context}"]
-= Subscribing the consumer application to PTP events
+[id="ptp-subscribing-consumer-app-to-events-{ptp-events-rest-api}_{context}"]
+= Subscribing to PTP events with the REST API {ptp-events-rest-api}
 
-Before the PTP events consumer application can poll for events, you need to subscribe the application to the event producer.
+ifeval::["{ptp-events-rest-api}" == "v2"]
+Deploy your `cloud-event-consumer` application container and subscribe the `cloud-event-consumer` application to PTP events posted by the `cloud-event-proxy` container in the pod managed by the PTP Operator.
 
-[id="ptp-sub-lock-state-events_{context}"]
-== Subscribing to PTP lock-state events
+Subscribe consumer applications to PTP events by sending a `POST` request to `http://localhost:9043/api/ocloudNotifications/v2/subscriptions` passing the appropriate subscription request payload.
 
-To create a subscription for PTP `lock-state` events, send a `POST` action to the cloud event API at `+http://localhost:8081/api/ocloudNotifications/v1/subscriptions+` with the following payload:
+[NOTE]
+====
+`9043` is the default port for the `cloud-event-consumer` container deployed in the application pod.
```

Comment:
`9043` is the default port for the `cloud-event-proxy` sidecar container deployed in the ptp producer pod.